lunes_cms/cms/admins/feedback_admin.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+from django.contrib.admin.views.decorators import staff_member_required
+
+from ..models import Feedback
+
+# @staff_member_required
+class FeedbackAdmin(admin.ModelAdmin):
+    """
+    Admin interface for feedback.
+    Inheriting from `admin.ModelAdmin`.
+    """
+
+    model = Feedback
+
```

Comment:
```suggestion
    list_display = readonly_fields = [
        "comment",
        "content_object_link",
        "content_object_type",
        "created_date",
        "read_by",
    ]
    search_fields = ["comment"]
    list_filter = ["content_type", "read_by"]
    sortable_by = ["content_object_type", "created_date", "read_by"]

    def has_add_permission(self, request, obj=None):
        return False

    def has_change_permission(self, request, obj=None):
        return False

    class Media:
        css = {"all": ("css/feedback.css",)}
```
This would be my suggestion for the feedback admin config, but you can have a look yourself whether you want to change the displayed fields etc...
Also, I think we can resolve the "read-only" part by just returning `False` for the add/change permissions...
And I'd like to add a bit of custom css to this list, since we don't know how much content the feedback entries might contain (and thus we don't know the table row width).